meta/labs_program/interview.md

```diff
@@ -0,0 +1,27 @@
+Explain what labs program is.
+
+0. How did your club go last year? What's your school's coding community like?
```

Comment:
Also, Markdown lists cannot start at 0. They always start at 1 when rendered. Can this be reordered to start at 1?